xml/vk.xml

```diff
@@ -143,7 +143,7 @@ server.
         <type requires="ggp_c/vulkan_types.h" name="GgpFrameToken"/>
 
         <type category="define">#define <name>VK_MAKE_VERSION</name>(major, minor, patch) \
-    (((major) &lt;&lt; 22) | ((minor) &lt;&lt; 12) | (patch))</type>
+    ((((uint32_t) major) &lt;&lt; 22) | (((uint32_t) minor) &lt;&lt; 12) | ((uint32_t) patch)))</type>
```

Comment:
In a macro, generally you want to be quite liberal with parentheses; it is text, not variables. E.g. `VK_MAKE_VERSION(1, 0, 1 + 1)` would expand to `(uint32_t)1 + 1`, which is not what you intended. Unlikely somebody uses it that way, but still...

Reply:
Done! I always get tricked by this scenarios with macros...